### PR TITLE
Remove Rails 6.0.z support

### PR DESCRIPTION
**Pull request by HeitorMC**
Removes Rails 6.0.z  support due to [EOL](https://guides.rubyonrails.org/maintenance_policy.html).

### CODE REVIEW

**Comment by mcmire** [comment]
@HeitorMC Ah, right. That makes sense!

**Comment by HeitorMC** [comment]
Hey, @mcmire @VSPPedro any updates on this case?

File: README.md [hunk 1]
```diff
@@ -27,25 +27,25 @@ complex, and error-prone.
...
-   * [ActionController matchers](#actioncontroller-matchers)
-   * [Independent matchers](#independent-matchers)
-* [Extensions](#extensions)
-* [Contributing](#contributing)
-* [Compatibility](#compatibility)
-* [Versioning](#versioning)
-* [Team](#team)
-* [Copyright/License](#copyright-license)
-* [About thoughtbot](#about-thoughtbot)
+- [Getting started](#getting-started)
```
Comment (mcmire) [inline README.md:30]: Are the formatting changes to the README necessary in this PR or can we do this in a different PR? Is there something that forced this change?

File: README.md [hunk 2]
```diff
@@ -125,8 +125,8 @@ Then run `bundle install`.
 
 Now you need to configure the gem by telling it:
 
-* which matchers you want to use in your tests
-* that you're using Minitest so that it can make those matchers available in
+- which matchers you want to use in your tests
```
Comment (VSPPedro) [inline README.md:128]: Why are we replacing * with _? I don't mind, just curious.

File: README.md (hunk 1 above)
Comment (HeitorMC) [reply README.md:30]: Oh, my bad. It was the linter :sweat_smile: Lemme fix it

File: README.md (hunk 2 above)
Comment (HeitorMC) [reply README.md:128]: This was my linter, my mistake. Lemme revert this heh

**Comment by VSPPedro** [comment]
Hi, @HeitorMC, LGTM! We only need to fix the conflict.
